ingestion/nse-scraper/.env.example

```
@@ -0,0 +1,40 @@
+TOPIC_TRADING_CALENDAR=reference.nse.trading_calendar
+
+# NSE Data Sources
+NSE_BHAVCOPY_URL=https://archives.nseindia.com/content/cm/BhavCopy_NSE_CM_0_0_0_{date}_F_0000.csv
+NSE_EQUITY_LIST_URL=https://archives.nseindia.com/content/equities/EQUITY_L.csv
+NSE_CA_URL=https://www.nseindia.com/api/corporates-corporateActions
+
+# Scraper Configuration
+SCRAPER_RETRY_ATTEMPTS=3
+SCRAPER_RETRY_DELAY=60
+SCRAPER_TIMEOUT=300
+SCRAPER_USER_AGENT=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)